multires_consensus_clustering/merge_resolution_graphs.py

```diff
@@ -290,6 +283,16 @@ def multires_community_detection(graph, clustering_data, community_detection, mu
         graph = reconnect_graph(graph)
 
         if graph.ecount() > 0:
+            # calculate max and min edge weights
+            max_edge = max(graph.es["weight"])
+            min_edge = min(graph.es["weight"])
+
+            # calculate max-min average
+            max_min_average = (max_edge + min_edge) / 2
+
+            # set the upper quantile for the merge edges threshold
+            merge_edges_threshold = (max_edge + max_min_average) / 2
+
```

Comment:
Could we use `np.percentile()` here as well?

Reply:
We could use no.percentile, but this would lead to a way higher threshold and thus more clusters (over-clustering).
Current threshold for the NeurIPS is ~75%
Np.percentile ~ 98%